arrows/pdal/pointcloud_io.h

```diff
@@ -22,12 +22,6 @@ namespace pdal {
 class KWIVER_ALGO_PDAL_EXPORT pointcloud_io
 {
 public:
-    /// Write landmarks to a file with PDAL provided a geo origin file
-    void
-    save_(vital::path_t const& filename,
-            vital::path_t const& input_geo_origin_file,
-            vital::landmark_map_sptr const& landmarks);
-
     /// Write landmarks to a file with PDAL
     void
     save_(vital::path_t const& filename,
```

Comment:
I didn't catch this before, but is there probably isn't a good reason for these functions to belong to a class.  They could just be stand alone functions.  There is no state that the class needs to maintain. 

Also we could just call these functions `save_landmarks_las` or `save_point_cloud_las` since they are specifically writing these data types to a LAS file.  The underscore suffix was used in image_io to distinuish between the pubic `save` function that is exposed and the private `save_` function that developer had to actually implement.  We don't need the underscore suffix here.